PDC Workspace:  Analysis(ID:${proteinSearchId}:"${proteinSearchName}") has been completed with errors
Hello, ${user}!

During analysis execution(${proteinSearchId}:"${proteinSearchName}") of "${experimentName}" Study created by you the error is occured.

Here is the details of this error:
Workflow step: ${failedWorkflowStep}
Error messsage:${errorMessage}

Please contact PDC Workspace Support Team to find out more.

Thank you!
